src/thunderbots_msgs/msg/DrawShape.msg

```diff
@@ -0,0 +1,27 @@
+# This message is for basic communication from AI to the visualizr
+# regarding shape drawing
+
+# Type of the shape
+# Can be one of: "ellipse" | "rect" | "poly" | "arc" | "line"
+string type
+
+# Data associated with the shape; expect different form depending on type:
+# If "ellipse": [center_x, center_y, r1, r2]
+# If "rect":   [x, y, width, height]
+# If "poly":   [x1, y1, x2, y2, ...]
+# If "arc":    [center_x, center_y, radius, theta_start, theta_end]
+# If "line":   [x1, y1, x2, y2]
+float64[] data
+
+# Fill of the shape
```

Comment:
I know that this string would be the name of a color from our previous discussions, but could we add a quick comment saying that here too?